doc/GLX_EXT_libglvnd.txt

```diff
@@ -0,0 +1,186 @@
+Name
```

Comment:
I'd be inclined to include the extension spec in glvnd; it seems likely to grow stale relative to the copy that will eventually be pushed to Khronos.